### DISCUSSION TITLE
How to install the stack on distroless

### DISCUSSION THREAD

**Discussion by bigmike09890** [strapi/strapi#7951] [Legacy Topics]
I'm attempting to get strapi installed to a distroless docker container but I'm not seeing really straightforward instructions on getting that stack set up for a server only environment.

I'm attempting to start with the nodejs distroless image from https://github.com/GoogleContainerTools/distroless/tree/master/nodejs 

Any type of guide available to install just the server without the development environment?

**Comment by derrickmehaffy** [comment]
We have the https://github.com/strapi/strapi-docker repo if may help you? I've never heard of distroless :thinking:

**Reply by plfriedman** [reply]
No, the point is to use [Distroless](https://github.com/GoogleContainerTools/distroless) to eliminate security vulnerabilities.  Basically, is there a way to get the Strapi artifacts directly to install in our own Distroless container?

**Reply by derrickmehaffy** [reply]
Not sure I understand what you mean by `Strapi artifacts` can you clarify a bit?

**Comment by soulscreme** [comment]
To add on here, what is the current maintenance cycle of the docker image that is published?  Currently it contains numerous CVEs.  One solution to prevent these CVEs (as most occur in unused Linux packages) is to move to distroless and run only with the things absolutely needed for Strapi.  However, the latest container is on node:11.1.0-alpine, this is a pretty old image as I believe they're up to 14.x at this point.

**Reply by plfriedman** [reply]
@derrickmehaffy this is the problem we are trying to solve, security scans prevent the use of the docker image because of CVE's exposed in the old image.

**Reply by derrickmehaffy** [reply]
In that case @plfriedman docker related issues and concerns should be made on the [strapi/strapi-docker](https://github.com/strapi/strapi-docker) repo as it's maintained differently than the core strapi/strapi repo.

**Reply by derrickmehaffy** [reply]
We don't currently have any information on distroless at the moment and it's not entirely an environment we support (as I've asked around the team and no one has heard of it)
